hana/primary_available.sls

```diff
@@ -0,0 +1,17 @@
+{%- from "hana/map.jinja" import hana with context -%}
+{% set host = grains['host'] %}
+
+{% for node in hana.nodes %}
+{% if node.host == host and node.secondary is defined %}
+netcat-openbsd:
+  pkg.installed
+
+primary-available:
+  cmd.run:
+    - name: until nc -z {{  node.secondary.remote_host }} 40002; do sleep 1; done
```

Comment:
@arbulu89 The logic here is a little bit more complex. See: https://help.sap.com/viewer/ports/616a3c0b1cc748238de9c0341b15c63c.html

The port calculation depends if HANA is running on a Single Container or Multi-Container installation, also depends on the instance number like `4<InsNr>02`.

Maybe @Simranpal knows a command or a metadata file to check the port in usage without needing to calculate that.

Reply:
True, I didn't notice this logic. Let's see if we can find a better solution to find the port.
Thx @diegoakechi

Reply:
As discussed, we will solve this with a different approach. Approving for now.